provider-service/kfp/internal/mock_metadata_store.go

```diff
@@ -30,8 +30,10 @@ func (mms *MockMetadataStore) reset() {
 func (mms *MockMetadataStore) returnArtifactForPipeline() []common.Artifact {
 	mms.results = []common.Artifact{
 		{
-			common.RandomString(),
-			common.RandomString(),
+			"artifact-name",
+			"artifact-location",
+			// common.RandomString(),
+			// common.RandomString(),
```

Comment:
```suggestion
```

Reply:
Missed this one yes, now in 5fa0ca3cdb7b98032ca0e4472a682e6195b7c436